frontend/src/features/idir/search/api/idirSearch.ts

```diff
@@ -15,7 +15,7 @@ const getSearchURLbyEntity = (searchEntity: SearchEntity): string | URL => {
       url = `${VEHICLES_URL}/${searchEntity}/ppc/search`;
     break;
     case SEARCH_ENTITIES.COMPANY:
-      url = `${VEHICLES_URL}/${searchEntity}/paginated`;
+      url = `${VEHICLES_URL}/${searchEntity}`;
```

Comment:
I think we can wait till #1109  is merged. This function could be reduced further or even removed.

Reply:
good point!